### PR TITLE
 Zmumu validation ported to 74X

### PR DESCRIPTION
**Pull request by cschomak**
Zmumu validation of all-in-one-tool ported to 74X

### CODE REVIEW

File: Alignment/OfflineValidation/python/TkAlAllInOneTool/zMuMuValidationTemplates.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 .oO[datasetDefinition]Oo.
 # process.load("Alignment.OfflineValidation..oO[dataset]Oo._cff")
 
-process.load("Geometry.CMSCommonData.cmsIdealGeometryXML_cfi")
-process.load("Geometry.CommonDetUnit.globalTrackingGeometry_cfi")
+#process.load("Geometry.CMSCommonData.cmsIdealGeometryXML_cfi")
+process.load("Geometry.CommonDetUnit.globalTrackingGeometryDB_cfi")
 process.load("RecoMuon.DetLayers.muonDetLayerGeometry_cfi")
```
Comment (mmusich) [inline Alignment/OfflineValidation/python/TkAlAllInOneTool/zMuMuValidationTemplates.py]: @cschomak is this inclusion really needed?

**Comment by mmusich** [comment]
please test

**Comment by mmusich** [comment]
@cschomak do you plan to fwd port to 76X and 75X too? We should have the usual approval procedure...

**Comment by cschomak** [comment]
Yes I plan to port to 76X and 75X too. Shall I wait with the PR in 74X until the others are done? So far I only looked at 74X.
On the inclusion: It might not be necessary, I will test it

**Comment by mmusich** [comment]
Just open sister PRs in all the active branches. Changes are trivial, we can approve everything in one go.

**Comment by cschomak** [comment]
The include statement is not necessary and I deleted it

**Comment by mmusich** [comment]
+1 
for https://github.com/cschomak/cmssw/commit/e65ee30c065239cb5a649a5ca7115cdd6935e131

**Comment by mmusich** [comment]
please test

**Comment by monttj** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**